packages/amplify_core/pubspec.yaml

```diff
@@ -14,6 +14,8 @@ dependencies:
   amplify_storage_plugin_interface: ^0.0.1-dev.2
   amplify_analytics_plugin_interface: ^0.0.1-dev.2
   amplify_auth_plugin_interface: ^0.0.1-dev.2
+  amplify_datastore_plugin_interface:
+    path: ../amplify_datastore_plugin_interface
```

Comment:
We should be able to remove this now that melos is in place.

Reply:
I somehow couldn't get it to work with melos. flutter pub get complains that  `amplify_datastore_plugin_interface` is not in the repository. I'll remove it from this PR though